Editor.md

```diff
@@ -25,4 +25,76 @@ Also mark the checkbox `[x] Enable Go Modules (vgo)` and `[x] Vendoring mode` un
 
 ## VSCode
 
-TODO
+Visual Studio Code is a lightweight but powerful source code editor which runs on your desktop and is available for Windows, macOS and Linux. It comes with built-in support for JavaScript, TypeScript and Node.js and has a rich ecosystem of extensions for other languages (such as C++, C#, Java, Python, PHP, Go) and runtimes (such as .NET and Unity).
+
+Using the Go extension for Visual Studio Code, you get language features like IntelliSense, code navigation, symbol search, bracket matching, snippets, and many more that will help you in Golang development.
+
+### Build, lint, and vet
+On save, the Go extension can run go build, go vet, and your choice of linting tool (golint or gometalinter) on the package of the current file.
+You can control these features via the settings below:
+```
+  go.buildOnSave
```

Comment:
nit: The extra leading spaces in code blocks will be rendered in the final result (see https://github.com/reddit/baseplate.go/blob/2927a4decfe2491f0da111ad8c9237ceb520804f/Editor.md#build-lint-and-vet for example), unless you mean it intentionally I think we should remove them. Same as the other code blocks.

Reply:
Updated them.